data-prepper-plugins/aws-lambda/src/main/java/org/opensearch/dataprepper/plugins/lambda/processor/LambdaProcessor.java

```diff
@@ -72,6 +73,7 @@ public class LambdaProcessor extends AbstractProcessor<Record<Event>, Record<Eve
     private final Counter numberOfRecordsFailedCounter;
     private final Counter numberOfRequestsSuccessCounter;
     private final Counter numberOfRequestsFailedCounter;
+    private final Counter lambdaResponseRecordsCounter;
```

Comment:
Should we use a `DistributionSummary` here? This will let us both count and understand the sizes per request.

Reply:
@dlvenable let's do this in the follow up PR